packages/design-system/src/components/index.js

```diff
@@ -79,3 +79,5 @@ export { Promo } from './Promo';
 export { UploadBar, FileUploadProgress } from './UploadBar';
 
 export { SearchBar } from './SearchBar';
+
+export { Icon, IconSize, IconType } from './Icon';
```

Comment:
You need to declare and export types for IconSize and IconType to make it accessible in TS projects